src/openrct2/world/TileInspector.cpp

```diff
@@ -122,14 +124,14 @@ sint32 tile_inspector_insert_corrupt_at(sint32 x, sint32 y, sint16 elementIndex,
 
         // Update the tile inspector's list for everyone who has the tile selected
         rct_window * const tileInspectorWindow = window_find_by_class(WC_TILE_INSPECTOR);
-        if (tileInspectorWindow != nullptr && (uint32)x == windowTileInspectorTileX && (uint32)y == windowTileInspectorTileY)
+        if ((uint32)x == windowTileInspectorTileX && (uint32)y == windowTileInspectorTileY)
```

Comment:
How come is the tileInspectorWindow check removed here, but is kept in tile_inspector_swap_elements_at and other actions?